roles/test_operator/tasks/cleanup.yml

```diff
@@ -25,6 +25,7 @@
     name: test-operator
     namespace: "{{ cifmw_test_operator_namespace }}"
     wait: true
+    wait_timeout: 600
```

Comment:
I was thinking about parametrizing this but I see that in ci-framework they use the 600 / 300 secs timeout without any parameter quite often (for example - [1]). 

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/7f82829499d69b5fbeead0a521530ffa29c1cd85/roles/kustomize_deploy/tasks/cleanup.yml#L35